app/components/provider_interface/application_header_components/offer_will_be_declined_by_default_component.rb

```diff
@@ -1,6 +1,16 @@
 module ProviderInterface
   module ApplicationHeaderComponents
     class OfferWillBeDeclinedByDefaultComponent < ApplicationChoiceHeaderComponent
+      def continuous_applications_offer_text
+        time_since_offer = days_from_now.zero? ? 'today' : "#{pluralize(days_from_now, 'day')} ago"
+
+        "You made this offer #{time_since_offer}. Most candidates respond to offers within 15 working days. The candidate will receive reminders to respond."
+      end
+
+      def days_from_now
+        (Time.zone.now.beginning_of_day - application_choice.offered_at.beginning_of_day).to_i / 1.day
```

Comment:
`application_choice.offered_at.beginning_of_day` should be ok as this component will only ever load if there is an offer